### PR TITLE
Changed to use example alerts

### PR DESCRIPTION
**Pull request by psiinon**
Part of https://github.com/zaproxy/zaproxy/issues/6119

Handles changes in https://github.com/zaproxy/zap-extensions/pull/2517

Signed-off-by: Simon Bennetts <psiinon@gmail.com>

### CODE REVIEW

File: scripts/generate_alert_pages.js [hunk 1]
```diff
@@ -4,8 +4,10 @@
 // The pages were created after starting a ZAP weekly release with the '-addoninstallall' option.
 
 // Change the DIR below to match the local directory containing the alert files
-var DIR = "/zap/wrk/zaproxy-website/site/content/docs/alerts/";
+// var DIR = "/zap/wrk/zaproxy-website/site/content/docs/alerts/"; TODO
+var DIR = "/Users/simon/workspace/zaproxy-website/site/content/docs/alerts/";
```
Comment (kingthorin) [inline scripts/generate_alert_pages.js:8]: Should be reverted?

Comment (psiinon) [reply scripts/generate_alert_pages.js:8]: Arg! yes...

Comment (psiinon) [reply scripts/generate_alert_pages.js]: Reverted.

**Comment by thc202** [comment]
Thanks!

**Pull request merged**